templates/vars/aks/aat.json

```diff
@@ -2,17 +2,20 @@
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
+    "nodeCount": {
+      "value": 16
+    },
     "nodeMinCount": {
-      "value": "16"
+      "value": ""
     },
     "nodeMaxCount": {
-      "value": "30"
+      "value": ""
     },
     "enableAutoScaling": {
-      "value": "true"
+      "value": ""
     },
     "type": {
-      "value": "VirtualMachineScaleSets"
+      "value": "AvailabilitySet"
```

Comment:
We shouldn’t go back to availability set, not sure if we need to go round everywhere disabling autoscaler, let’s wait for Microsoft, I assume we can build a cluster without it and then enable after as well

Reply:
I wanted to test that but didn't get time yesterday.